<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="prev_page">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>440</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="friend_1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>好友1</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="friend_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>130</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>好友2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="friend_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>190</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>好友3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="friend_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>250</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>好友4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="friend_5">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>310</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>好友5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="friend_7">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>130</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>好友7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="friend_6">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>70</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>好友6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="friend_10">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>310</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>好友10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="friend_8">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>190</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>好友8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="friend_9">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>250</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>好友9</string>
    </property>
   </widget>
   <widget class="QLabel" name="friend_label">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>你的好友：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>431</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>查找好友</string>
    </property>
   </widget>
   <widget class="QLabel" name="user_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>42</pointsize>
     </font>
    </property>
    <property name="text">
     <string>功能：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next_page">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>440</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>下一页</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>-1</y>
      <width>931</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:2, stop:0 rgba(255, 152, 236, 255), stop:1 rgba(255, 255, 255, 255))</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="viewApplyButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>431</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>查看申请</string>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>prev_page</zorder>
   <zorder>friend_1</zorder>
   <zorder>friend_2</zorder>
   <zorder>friend_3</zorder>
   <zorder>friend_4</zorder>
   <zorder>friend_5</zorder>
   <zorder>friend_7</zorder>
   <zorder>friend_6</zorder>
   <zorder>friend_10</zorder>
   <zorder>friend_8</zorder>
   <zorder>friend_9</zorder>
   <zorder>friend_label</zorder>
   <zorder>searchButton</zorder>
   <zorder>user_label</zorder>
   <zorder>next_page</zorder>
   <zorder>viewApplyButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
